benchmark/CMakeLists.txt

```diff
@@ -21,9 +21,10 @@ set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
 include(CheckCXXCompilerFlag)
 
 string(TOUPPER "${CMAKE_BUILD_TYPE}" U_CMAKE_BUILD_TYPE)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Ofast -march=native -Wunused-parameter -Wextra -Wreorder -Wconversion")
```

Comment:
You should not do this.

- It is redundant with the other directive below.
- It will likely break the build on compilers which do not use GCC's flag syntax (like MSVC).

Reply:
Use target_compile_options and generator to handle that properly

```c++
    target_compile_options(frozen.tests.html_entities_map PUBLIC
        $<$<CXX_COMPILER_ID:Clang>:     -fconstexpr-steps=123456789>
        $<$<CXX_COMPILER_ID:AppleClang>:-fconstexpr-steps=123456789>
    )
```

Oh, and ``-0fast`` is also called ``-Ocheating`` :-)